### DISCUSSION TITLE
Roadmap

### DISCUSSION THREAD

**Discussion by gaffinet** [sepinf-inc/IPED#536] [Q&A]
Hi 

Does this project has a roadmap?

The reason I'm asking this question, is that I will start working on a huge case and if a new version (perhaps 4.0) will be release soon, I would wait till the release.

**Comment by lfcnassif** [comment]
We have some informal roadmap. For version 4.0.0 at least #23, #24, #538 and #232 are planned. Those will need maybe 2-3 months of work.

But I **plan** to publish a 3.18.6 bug fix release this week.
